includes/utils.php

```diff
@@ -90,6 +90,20 @@ class Utils {
 		],
 	];
 
+	/**
+	 * Variables for free to pro upsale modal promotions
+	 */
+
+	const ANIMATED_HEADLINE = 'animated_headline';
+
+	const CTA = 'cta';
+
+	const VIDEO_PLAYLIST = 'video_playlist';
+
+	const TESTIMONIAL_WIDGET = 'testimonial_widget';
+
+	const IMAGE_CAROUSEL = 'image_carousel';
```

Comment:
Should these not be in the Promotions module?

Reply:
I'd love that, but the usage is not contained by the `promotions` module alone.
a.t.m. it is used in:

```
   modules
   `- promotions
   includes
   `- widgets
      `- heading.php
      `- button.php
      `- ...
      `- testimonial.php
```

[ + ] potentially the spread-area will grow to other widgets, if marketing will find this new info-tip more attractive...
that's why we need a more "shared" space for that...